screens/UserTimeCards.js

```diff
@@ -0,0 +1,170 @@
+import { ScrollView, Text, View, TouchableOpacity, Image, ActivityIndicator, FlatList, Modal } from 'react-native'
+import { DismissKeyboard } from '../components/DismissKeyboard'
+import BackBtn from '../components/BackBtn'
+import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome'
+import Icon from 'react-native-vector-icons/Ionicons'
+import { getTimeCards, getUserTimeCards, formatDate, getMonthName, formatDateObj } from '../actions'
+import images from '../assets/images'
+import styles from '../styles'
+import { LocaleContext } from '../locales/LocaleContext'
+import DropDown from '../components/DropDown'
+import UserTimeCardsFilterForm from './UserTimeCardsFilterForm'
+import { api, dispatchFetchRequest, errorAlert, warningMessage } from '../constants/Backend'
+
+class UserTimeCards extends React.Component {
+  static navigationOptions = {
+    header: null
+  }
+  static contextType = LocaleContext
+
+  state = {
+  	timecardId: null,  	
+  	filteredUsrTimeCards: []
+  }
+
+  handleFilter = (values) => {
+  	var month = values.month;
+  	var year = values.year;
+  	var username = this.props.navigation.state.params.name;
+
+  	if (!month || !year) {
+      warningMessage('Please Choose Both Year and Month')
+      return
+    }
+		
+     dispatchFetchRequest(api.timecard.getByMonthYrUsr(year, month, username), {
+        method: 'GET',
+        withCredentials: true,
+        credentials: 'include',
+        headers: {}
+      },
+      response => {
+        response.json().then(data => {
+          this.setState({filteredUsrTimeCards: data.timeCards})
+        })
+      }).then()
+  }
+
+  componentDidMount() {
+  	this.props.getUserTimeCards(this.props.navigation.state.params.name)
+  	var defaultVal={
+  		year: new Date().getFullYear(),
+  		month: getMonthName(new Date().getMonth() + 1),
+  		username: this.props.navigation.state.params.name
+  	}
+  	this.handleFilter(defaultVal)
+  }
+
+  render() {
+    const { t } = this.context
+    const { usertimeCards, haveData, haveError, loading, timeCard } = this.props
+    const { filteredUsrTimeCards } = this.state
+
+    Item = ({ timecard }) => {
+      return ( 	
+      	<TouchableOpacity
+					onPress={() => {
+          	this.props.navigation.navigate('UserTimeCardDetail',{
+          		timecardId: timecard.id
+          	})
+          }}
+      		>  
+        	<View style={[{paddingHorizontal: 8, marginBottom: 10}]}>
+          	<View style={[styles.flex_dir_row, styles.paddingTopBtn8]}>
+            	<View style={{flex: 6}}>
+              	<Text>
+              		{formatDateObj(timecard.clockIn)}&nbsp;&nbsp;{new Date(timecard.clockIn).getUTCHours() + ':' + new Date(timecard.clockIn).getUTCMinutes()} 
+              		{new Date(timecard.clockIn).getUTCHours() <= 12 ? 'am' : 'pm'}
+              	</Text>
+            	</View>
+
+            	<View style={{flex: 4}}>
+              	<Text style={{textAlign: 'right'}}>
+              	{
+              		timecard.hours > 1  
```

Comment:
don't need to check for hours > 1. Just display hours and minutes.

Like {timecard.hours} t('hours') {timecard.minutes} t('minutes')